tests/unit/banners/test_views.py

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date, timedelta
+
+from warehouse.banners import views
+
+from ...common.db.banners import BannerFactory
+
+
+def test_list_active_banners(db_request):
+    today = date.today()
+    ten_days = timedelta(days=10)
+
+    active_baner = BannerFactory.create()
+    # past banner
+    BannerFactory.create(
+        begin=today - (ten_days * 2),
+        end=today - ten_days,
+    )
+    # future banner
+    BannerFactory.create(
+        begin=today + ten_days,
+        end=today + (ten_days * 2),
+    )
+
+    result = views.list_banner_messages(db_request)
+
+    assert len(result["banners"]) == 1
+    assert result["banners"][0] == active_baner
```

Comment:
```
assert result["banners"] == [active_baner]
```
(also, banner with 2 n)

Reply:
Thanks for catching the typo! I'll update the tests

Reply:
Changed in b410eea2d7187f16ba87af84387f08935a46df3b

Reply:
This wasn't a typo, just a way to make the 2 asserts in one, in a way that is more informative if it fails.

I've seen you've replaced the 2nd line, but the one just above is useless now, we explicitely check that the list has one element by comparing it to a list that has 1 element :)

Reply:
Yeah, this is a clever way to test this, for sure! Removed the `len` assertion in de61b0b99392e4ce616191cc99e2e8674529f7b4

Ps.: the typo was in previous commits. The variable was named `active_baner`, with a single `n`.